### DISCUSSION TITLE
Styleguide da comunidade?

### DISCUSSION THREAD

**Discussion by celsomtrindade** [frontendbr/forum#514] [Discussão]
Bom pessoal, a algum tempo eu tenho participado aqui do fórum, mas nunca fui ativo/interativo. Hoje fiquei dando uma boa olhada e buscando algo que pudesse me envolver mais. Como estava remodelando/revisando o _styleguide_ que uso (uso pessoal) resolvi pesquisar aqui para buscar me atualizar, buscar novas opiniões, etc.

Uma coisa que notei foi que, não existe uma _styleguide_ da comunidade, ou pelo menos eu não encontrei. O máximo que encontrei foram esses tópicos #129 e #182  abordando o assunto, porém sempre direcionando para algum local externo.

Sei que é algo muito pessoal ou que muitas vezes não podemos mudar, pois está atrelado a um ambiente de trabalho já consagrado. Mas também sei que muita gente pode usar e se basear parar criar um novo _styleguide_, ou até mesmo atualizar um já existente.

Como temos muitos membros e muita experiência em conjunto!!!!! Acho que vale a pena dedicarmos uma área para isso. O que acham?

Baseado nessa ideia, eu publiquei o guia que utilizo nos meus projetos em meu github e, se a ideia for interessante, podemos usar como ponto de partida (ou utilizar algum outro - A ideia é ter algo que possamos elaborar juntos). [Link do meu _styleguide_](https://github.com/celsomtrindade/CSS-Styleguide).

O que acham?

**Comment by cristofersousa** [comment]
Celso, esses meses estive exatamente com essa preoucupação em criar um styleguide para a empresa que trabalho, vasculhando por aí, achei algumas referencias para sair do zero: 

1. [styleguides.io](http://styleguides.io/)
2. [cssguidelin.es](http://cssguidelin.es/)
3. [huge styleguide](http://hugeinc.github.io/styleguide/es/start.html)
4. [air bnb](https://github.com/airbnb/css)

[],s

**Comment by celsomtrindade** [comment]
@cristofersousa Sim! Eu mesmo utilizei diversos styleguides antes de montar um de uso próprio. Como a comunidade, que ajuda tanto uns aos outros, com opiniões, troca de experiência e muito mais, ainda não possui algo no estilo, achei que seria bacana propor essa ideia. É algo que todos usamos (ou deveríamos utilizar) e que acredito que possa agregar bastante.
